rholang/examples/bond/bond.rho

```diff
@@ -0,0 +1,8 @@
+new retCh, PoSCh, rl(`rho:registry:lookup`), stdout(`rho:io:stdout`) in {
+  stdout!("About to lookup pos contract") |
+  rl!(`rho:rchain:pos`, *PoSCh) |
+  for(@(_, PoS) <- PoSCh) {
+    stdout!("About to bond") |
+    @PoS!("bond", 100, *retCh)
```

Comment:
You should wait for the bond result on retCh and display a "bond successful message" or a "bond failed" message

Reply:
PTAL